ui/components/multichain/account-overview/account-overview-tabs.tsx

```diff
@@ -156,7 +156,7 @@ export const AccountOverviewTabs = ({
             data-testid="account-overview__activity-tab"
             {...tabProps}
           >
-            <TransactionList boxProps={{ paddingTop: 3 }} />
+            <TransactionList />
```

Comment:
Removed extra padding so that the activity bar is aligned across pages without any extra hacks.